### PR TITLE
Upload photo

### PR DESCRIPTION
**Pull request by desolee**

### CODE REVIEW

File: flinder/flinderserver/views.py [hunk 1]
```diff
@@ -140,12 +141,19 @@ def register_room_provider(request):
...
-    response = render(request, "flinder/upload_photos.html")
-
-    return response
+    # Query the database for any data needed to build the page
+    if request.method == 'GET':
+        imgs = Pictures.objects.get(pk=1)
+        return render(request, 'flinder/upload_photos.html', {'imgs': imgs})
+    elif request.method == 'POST':
+        print(request)
+        imgfiles = request.FILES.get('img')
```
Comment (desolee) [inline flinder/flinderserver/views.py:152]: I can't receive img, 
image

File: flinder/flinderserver/urls.py [hunk 2]
```diff
@@ -13,7 +14,7 @@
     path('register/account_type', views.register_account_type, name="register_account_type"),
     path('register/room_seeker', views.register_room_seeker, name="register_room_seeker"),
     path('register/room_provider', views.register_room_provider, name="register_room_provider"),
-    path('upload_photos', views.upload_photos, name="upload_photos"),
+    path('upload_photos', csrf_exempt(views.upload_photos), name="upload_photos"),
```
Comment (space928) [inline flinder/flinderserver/urls.py:17]: Why is this csrf exempt?

File: flinder/flinderserver/views.py [hunk 3]
```diff
@@ -140,12 +141,19 @@ def register_room_provider(request):
...
+        img = Pictures(picture=imgfile, description=imgfile.name)
```
Comment (space928) [inline flinder/flinderserver/views.py:153]: typo in imgfile should be imgfiles

File: flinder/templates/flinder/upload_photos.html [hunk 4]
```diff
@@ -21,9 +21,14 @@ <h1 class="jumbotron-heading my-4 col-12 col-md-6">Upload photos</h1>
...
-                <div class="text-center h1 lh-100">+</div>
-            </div>
+{#            <div class="image-upload-box rounded" ondragover="return false;" ondrop="updateImage(event)" onclick="openFileBrowser()">#}
+{#                <div class="text-center h1 lh-100">+</div>#}
+{#            </div>#}
+            <form action="{% url 'flinder:upload_photos' %}" method="post" enctype="multipart/form-data">
+                {% csrf_token %}
+                <input name="img" type="file">
+                <button class="btn btn-primary w-auto my-1" type="submit">Upload</button>
+            </form>
```
Comment (space928) [inline flinder/templates/flinder/upload_photos.html:31]: I'm currently working on updating the HTML template, so I'll reject your changes on the html.

**Review by space928 — CHANGES_REQUESTED** [review]

File: flinder/flinderserver/views.py (hunk 3 above)
Comment (desolee) [reply flinder/flinderserver/views.py:153]: Oh, thank you I didn't notice that

File: flinder/flinderserver/urls.py (hunk 2 above)
Comment (desolee) [reply flinder/flinderserver/urls.py:17]: Just for testing, I will remove this, I use postman to have a test

File: flinder/templates/flinder/upload_photos.html (hunk 4 above)
Comment (desolee) [reply flinder/templates/flinder/upload_photos.html:31]: OK, as long as it post the img to the backend, then that's fine

**Review by space928 — APPROVED** [review]

**Pull request merged**